portfolio/src/main/webapp/index.html

```diff
@@ -11,6 +11,7 @@
     <div class = "flex-nav-bar">
         <a href="index.html" > <p class="nav-item bold"> Home </p> </a>
```

Comment:
Do you need the empty spaces between tags?

Reply:
No, I don't. Will remove the empty spaces. Resolved